src/filtered-list/index.ts

```diff
@@ -0,0 +1,67 @@
+import { tag, Tag } from '@storefront/core';
+import { DEFAULT_INDEX_ALIAS, DEFAULT_ITEM_ALIAS } from '../list';
+
+@tag('gb-filtered-list', require('./index.html'))
+class FilteredList {
+
+  refs: { filter: HTMLInputElement };
+  items: FilteredList.Item[];
+  originalItems: FilteredList.Item[];
+  props: FilteredList.Props = {
+    items: [],
+    itemAlias: DEFAULT_ITEM_ALIAS,
+    indexAlias: DEFAULT_INDEX_ALIAS,
+  };
+
+  onMount() {
+    this.originalItems = this.props.items;
+    this.updateItems();
```

Comment:
Actually, just copy all the items in directly. There should be no filter when the component is first mounted.